src/clojure/funcatron/tron/util.clj

```diff
@@ -71,18 +73,26 @@
   )
 
 
-
+(s/fdef kwd-to-string
+  :args (s/cat :kw? keyword?)
+  :ret string?)
 (defn kwd-to-string
-  "Converts a keyword to a String"
+  "Converts a keyword to a nsless String"
   [kw?]
-  (if (keyword? kw?) (name kw?) kw?)
-  )
+  (if (keyword? kw?) (name kw?) kw?))
 
+(stest/instrument 'kwd-to-string)
```

Comment:
This will turn on instrumenting in production and thus **really** slow the app down. Please remove the `stest` namespace from here and only use it in the test code.